### DISCUSSION TITLE
all_block_device_type_should_be_converted_from_or_into_i32 TC 문의

### DISCUSSION THREAD

**Discussion by dhihm** [huijeong-kim/ministore#14] [Q&A] [answered]
block_device_common::tests::all_block_device_type_should_be_converted_from_or_into_i32 이 테스트 케이스가 fail 이라서 

잠깐 살펴 봤는데,  BlockDeviceType enum에서 io uring이 삭제 되어 있어서, 지금 값이 안 맞는 것 같네요. match 시에 async를 1로 해야 할듯?

<img width="1027" alt="image" src="https://user-images.githubusercontent.com/28741816/236435158-be22352f-5abe-4221-ad2b-697a1233beb7.png">

**Comment by huijeong-kim** [comment]
오 그렇네요! 수정하겠습니다~~!! 추가는 항상 마지막에 하도록 해야겠군요.

**Comment by huijeong-kim** [comment, marked as answer]
아 테스트 코드 이슈는 Issues에 올려주시거나 직접 PR 해주셔도 좋을 것 같습니다...!! 감사함당! :)

**Reply by dhihm** [reply]
contribution의 기회가!!!!!

**Reply by huijeong-kim** [reply]
헛.. 이래놓고 제가 PR을 올려버렸군요.. 다음에...
